### ISSUE TITLE
julia syntax outdated

### ISSUE THREAD

**Issue by xianhegithub** [christianpeel/LDPCsim.jl#1]
Hi Chris, 
I'm trying your ldpc sim in julia, but the syntax is outdated due to the rapid update of julia. Do you remember how long it takes to run each simulation? I somehow modified the code according to the new julia syntax and made them run, but it is taking ages.

**Comment by chrisvwx** [comment]
Yes, I do remember it being very slow.  I do think it would be a good thing to have fast LDPC design/encoding/decoding code, yet this code is not it.  I never did fully learn and internalize LDPC codes enough to make this a useful tool. If I recall correctly this code is a fairly straightforward port of some [matlab code](https://www.mathworks.com/matlabcentral/fileexchange/14869-ldpc-codes-ber-simulation).    

The following disclaimer is in the README
> This was implemented as a learning excercise and is almost certainly not yet widely useful. The "sim" in "LDPCsim" at present stands for both "simple" and "simulate" :-)

I'll update this to indicate that (unfortunately) I don't plan to update this code. Of course you're welcome to take it and improve it as you see fit.
